org.ekstep.question-1.0/editor/question-ctrl.js

```diff
@@ -240,6 +240,9 @@ angular.module('org.ekstep.question', ['org.ekstep.metadataform'])
   }
   $scope.sendMetaData = function (newQuestionCreate) {
     $scope.isNewQuestion = newQuestionCreate;
+    if($scope.isNewQuestion === false && EventBus.hasEventListener('org.ekstep.questionunit:ready')){
```

Comment:
Why are removing the listeners? you have to clear the form fields scope object.

Reply:
You can do this on save or create new.

Reply:
This is what i observed:
1.When we are creating the question for first time 'org.ekstep.questionunit:ready' event listener is not there.
2. when we edit a question 'org.ekstep.questionunit:ready' listener is getting added and not removed after submit ,so when we create a new question after editing the old data is getting added because of event listener.Thats why i removed the event listener.